tests/server/test_auth_config.py

```diff
@@ -0,0 +1,25 @@
+from sams.factory import get_app
```

Comment:
in the future when there will be a lot of test files, we can move this into subfolder:
`tests/server/auth/`